### DISCUSSION TITLE
Thanks <3

### DISCUSSION THREAD

**Discussion by rix1** [shuding/nextra#324] [General]
With COVID running rampant I decided to spend my Friday evening in front of my computer. I wanted to test out `Nextra@alpha`, and after some `pnpm` hurdles[1], I got everything working 🎉 

Everything looked great (seriously, the new theme is :chef-kiss:). However, I discovered the theme switcher was a bit off. I traced it down to a `.menu` CSS class so I figured – what the hell, I might as well open a PR. This is how open source works, right?

<img width="317" alt="Screenshot by Rix1 2022-01-14 at 22 04 09" src="https://user-images.githubusercontent.com/2470775/149586940-02d074da-55b3-4c00-8215-e5d34e916b82.png">

So, I cracked my fingers, put on some Drum&Bass and cloned the repo. 

_[Entering the Matrix]_

1m28s seconds later, I discovered the CSS class issue was already fixed upstream by @shu a couple of days ago.

_[Exiting the Matrix]_

So instead of actually fixing anything, I ended up writing this post instead. Just wanted to say thanks for making awesome stuff and fixing things faster than I can clone a repo @shuding 🙌 ❤️ 

---

_[1]: That command is seriously putting my muscle memory to the ultimate test... I have now aliased `pnpnm`, `npmp`, `npnmp`, `pmpm` and `[mpnp` to simply `$> pp` – sorry [PAR packager](http://manpages.ubuntu.com/manpages/bionic/man3/pp.3pm.html) 🙃_

**Comment by shuding** [comment]
Thanks a lot @rix1, this made my day!

> I have now aliased pnpnm, npmp, npnmp, pmpm

Lol I'm going to add this too...
